include/network/server.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2016 CodiLime
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef VELES_NETWORK_SERVER_H
+#define VELES_NETWORK_SERVER_H
+
+#include <QtNetwork/QTcpServer>
+
+#include "db/types.h"
+#include "network.pb.h"
+
+namespace veles {
+namespace db {
+
+class NetworkServer : public QObject {
```

Comment:
Currently we use Qt on "server" side (and thanks to it I didn't have to worry about portability of network code) so I decided to go with it for now.